flink-dist/src/main/flink-bin/bin/pyflink-gateway-server.sh

```diff
@@ -63,7 +63,7 @@ if [[ -n "$FLINK_TESTING" ]]; then
     else
       FLINK_TEST_CLASSPATH="$FLINK_TEST_CLASSPATH":"$testJarFile"
     fi
-  done < <(find "$FLINK_SOURCE_ROOT_DIR" ! -type d -name 'flink-*-tests.jar' -print0 | sort -z)
+  done < <(find "$FLINK_SOURCE_ROOT_DIR" ! -type d \( -name 'flink-*-tests.jar' -o -path '*flink-connector-*-base/target/flink*.jar' \) -print0 | sort -z)
```

Comment:
Specify flink-connector-kafka-base and flink-connector-elasticsearch-base explicitly as flink-connector-*-base is not a good pattern and these two connectors just happen to have this pattern.

Reply:
Done.